src/nvim/ex_cmds.h

```diff
@@ -31,30 +32,19 @@ typedef struct {
 
 // Defs for inc_sub functionality
 
-// List of matches in a line
-#define _noop(x)
-KLIST_INIT(colnr_T, colnr_T, _noop)
-typedef klist_t(colnr_T) klist_colnr_T;
-#undef _noop
+// List of matches on a single line
+typedef kvec_t(colnr_T) kvec_colnr_T;
```

Comment:
It should be `ColNrVec` according to the style guide, not `…_T`.

@bfredl Using `kvec_t(colnr_T)` is not a good idea, you can’t pass `kvec_t(colnr_T) *foo` to `void bar(kvec_t(colnr_T) foo)`:

```
echo $'struct { int a; } *foo; int main(int argc, char **argv, char **environ){ struct { int a; } bar; foo = &bar; }' | clang -Wall -pedantic -std=c99 -xc -
<stdin>:1:101: warning: incompatible pointer types assigning to 'struct (anonymous struct at <stdin>:1:1) *' from 'struct (anonymous struct at <stdin>:1:74) *' [-Wincompatible-pointer-types]
struct { int a; } *foo; int main(int argc, char **argv, char **environ){ struct { int a; } bar; foo = &bar; }
                                                                                                    ^ ~~~~
1 warning generated.
```